### ISSUE TITLE
Project property page "HelpUrl" icon is not keyboard accessible

### ISSUE THREAD

**Issue by japarson** [dotnet/project-system#7998]
## Visual Studio Version
Microsoft Visual Studio Enterprise 2022 Preview
Version 17.2.0 Preview 1.0

## Summary
The project property pages contain a help icon which allows the user to navigate to a web page that contains more information about certain properties. This help icon is not keyboard focusable, which is an accessibility issue.
- [Bug 1504380](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1504380): A11y_Visual Studio Client_Android options_Enable trimming_Keyboard : "?" hyperlink icon present in the "Enable trimming" section is not accessible using keyboard.

https://user-images.githubusercontent.com/59936622/159573714-cffb7237-8178-47d3-9ea5-76aa888496b7.mp4

## Steps to Reproduce
1. Create new Android project from power shell using `dotnet new android -n <project_name>`.
1. Open VS 2022 and tab till "Open a project or solution" option and press enter.
1. Tab till "File name" and select newly created iOS "csproj" file and press enter on "Open" button.
1. Tab till project in "Solution explorer" and press context key.
1. Tab till "Properties" menu item and press enter.
1. Tab till "Android Options" button and press enter.
1. Tab till "Enable trimming" section.
1. Tab till "?" hyperlink icon.
1. Check whether the "?" hyperlink icon is accessible using keyboard or not.

## Expected Behavior
The help icon should be keyboard accessible.

## Actual Behavior
The help icon is not keyboard accessible.

## User Impact
Keyboard users may feel inconvenient in accessing "?" hyperlink icon if it's not accessible using keyboard.

[MAS 2.1.1 – Keyboard](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7b1F29ACF9-A3E7-414F-BD8D-7E3598D877BB%7d&file=MAS%202.1.1%20%E2%80%93%20Keyboard.docx&wdLOR=c6FA267E7-81CB-46D6-A357-F39B5ABB492F&action=default&mobileredirect=true)

**Comment by drewnoakes** [comment]
To open help on a property that shows the help icon, press <kbd>F1</kbd> while the editor for that property is focussed.

The help icon is intentionally not a tab stop, as it interferes to navigation, and <kbd>F1</kbd> is an established pattern for help in VS.

**Comment by tmeschter** [comment]
@melytc Feel free to close if this behavior is by design.
